extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
```

Comment:
```suggestion
```

You can remove this header!

Reply:
Ah ok! I had thought we left it in for future unreleased changes to be added (seems like sometimes it was left in in the past and sometimes not: https://github.com/github/vscode-codeql/pull/1277)

Reply:
Ah cool 😄